### DISCUSSION TITLE
ww3_from_ftp.sh is not being able to download files from ftp://polar.ncep.noaa.gov/tempor/ww3ftp/

### DISCUSSION THREAD

**Discussion by rafacsantana** [NOAA-EMC/WW3#1022] [Q&A] [answered]
Hi,

I've recently downloaded WW3 code from the git repo and I'm following the manual to set up and run the model. When I run ./ww3_from_ftp.sh the download of the .tar.gz file cannot be completed (log below). Does anyone have an idea to solve that or would be able to contact the manager of that ftp, please? 

Kind regards,
Rafa

>>

 This script will download data from the ftp for WAVEWATCH III
Enter the relative path to the main/top level directory, this would
be '../../' if in the model/bin directory or '.' if already in the
top/main directory:
../../

Downloading and untaring file from ftp:
--2023-06-16 15:40:45--  ftp://polar.ncep.noaa.gov/tempor/ww3ftp/ww3_from_ftp.v6.07.tar.gz
           => ‘ww3_from_ftp.v6.07.tar.gz’
Resolving polar.ncep.noaa.gov (polar.ncep.noaa.gov)... 140.90.101.132
Connecting to polar.ncep.noaa.gov (polar.ncep.noaa.gov)|140.90.101.132|:21... connected.
Logging in as anonymous ... Logged in!
==> SYST ... done.    ==> PWD ... done.
==> TYPE I ... done.  ==> CWD (1) /tempor/ww3ftp ...
No such directory ‘tempor/ww3ftp’.

tar (child): ww3_from_ftp.v6.07.tar.gz: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
Moving data from data_regtests to regtets

**Comment by MatthewMasarik-NOAA** [comment, marked as answer]
Hi @rafacsantana, it looks like you're using an older version of the code.  Currently both the ftp URL and bundle number have been updated from what you show in your output.  Please update your version of WW3, then you should be successful doing the following:

```
cd WW3
./model/bin/ww3_from_ftp.sh
```

then you should see output similar to this

```
[MM@hfe06 WW3(develop)]$ ./model/bin/ww3_from_ftp.sh 


 This script will download data from the ftp for WAVEWATCH III 
/scratch1/NCEPDEV/climate/Matthew.Masarik/projs/code_mgmt/work/WW3
Downloading and untaring file from ftp:
--2023-06-20 14:06:14--  https://ftp.emc.ncep.noaa.gov/static_files/public/WW3/ww3_from_ftp.v7.14.1.tar.gz
Resolving ftp.emc.ncep.noaa.gov (ftp.emc.ncep.noaa.gov)... 140.90.101.74
Connecting to ftp.emc.ncep.noaa.gov (ftp.emc.ncep.noaa.gov)|140.90.101.74|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 341842409 (326M) [application/x-gzip]
Saving to: ‘ww3_from_ftp.v7.14.1.tar.gz’

100%[=======================================================================>] 341,842,409 5.69MB/s   in 63s    

2023-06-20 14:07:18 (5.17 MB/s) - ‘ww3_from_ftp.v7.14.1.tar.gz’ saved [341842409/341842409]

data_regtests/
data_regtests/ww3_ufs1.1/
    .
    .
(truncated)
    .
    .
data_regtests/ww3_ufs1.2/input/gnh_10m.bot
data_regtests/ww3_ufs1.2/input/aoc_9km.obs
Moving data from data_regtests to regtests


 Do you want to delete the tar file ww3_from_ftp.v7.14.1.tar.gz [y|n]: 
n

 Deleting tar file ww3_from_ftp.v7.14.1.tar.gz


 Files were copied from the data_regtests to the regtests folder.
Do you want to delete the data_regtests folder? [y|n]: 
n

 Deleting the data_regtests folder
End of ww3_from_ftp.sh

```

**Reply by rafacsantana** [reply]
Thank you @MatthewMasarik-NOAA. It worked. Cloning https://github.com/NOAA-EMC/WW3 (without .git - as in the manual) downloaded the older version.

**Reply by MatthewMasarik-NOAA** [reply]
@rafacsantana I'm glad to hear the issue is resolved.
